custom_components/storj/backup.py

```diff
@@ -101,14 +102,16 @@ async def async_download_backup(
         self,
         backup_id: str,
         **kwargs: Any,
-    ) -> AsyncIterator[bytes]:
+    ):
```

Comment:
I will need to bring this back when I actually implement the download function. I'm surprised that I'm able to do this and still satisfy mypy since this function is needed because we're extending from an abstract class that defines it.